praw/models/reddit/live.py

```diff
@@ -281,6 +281,55 @@ def close(self):
         url = API_PATH['live_close'].format(id=self.thread.id)
         self.thread._reddit.post(url)
 
+    def update(self, **settings):
```

Comment:
How about this?

```
def update(self, title, description='', nsfw=False, resources='', **other_settings)
```

This siginature doesn't maintain current settings for unspecified settings (and reddit does the same thing), but eliminates citrical section problem as you pointed out in other comment.